packages/components/.storybook/web/manager.ts

```diff
@@ -0,0 +1,11 @@
+import { addons } from '@storybook/manager-api'
+
+addons.setConfig({
```

Comment:
Not sure if we want to clean this up now that it's no longer used. Seems handy to have around if we need it, but not sure if we'll ever use it again.